DataConnectors/O365 DataCSharp/Teams.CustomConnector.Common/Teams.CustomConnector.Common.csproj

```diff
@@ -5,7 +5,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.KeyVault" Version="3.0.5" />
+    <PackageReference Include="Azure.Identity" Version="1.2.3" />
```

Comment:
I highly recommend 1.4.1. There's been a lot of fixes and improvements since 1.2.3.

Reply:
@heaths - I have updated, please review